packages/actor-rdf-resolve-hypermedia-qpf/lib/ActorRdfResolveHypermediaQpf.ts

```diff
@@ -29,7 +30,26 @@ implements IActorRdfResolveHypermediaQpfArgs {
       throw new Error(`${this.name} requires metadata and searchForms to work on.`);
     }
 
-    return true;
+    // Find a quad pattern or triple pattern search form
+    const searchForms: ISearchForms = action.metadata.searchForms;
+    
+    // TODO: in the future, a query-based search form getter should be used.
```

Comment:
Is this the reason for #415? Should we immediately do it right here?

Reply:
I'm afraid this TODO is not that trivial to resolve, so I would leave it for a separate PR.